rel-eng/packages/caasp-management-node-formula

```diff
@@ -0,0 +1 @@
+4.1.7-1 susemanager-utils/cluster-providers/caasp/formulas/caasp-management-node-formula
```

Comment:
As discussed on rocket.chat, the three files from `rel-eng/packages` can go away. Together with that, remove the headers from the changelogs, so you only have the changelog entry (no empty lines).

If you need it for now, just remember to do it before merging, or tell me to do it after merging.

**Do NOT run `tito tag`!!!**

Reply:
Done now